Dockerfile.build

```diff
@@ -34,6 +34,8 @@ COPY ./package.json ./package.json
 COPY --from=protoc-build ./graphhopper/grpc ./grpc
 RUN npm install
 RUN npm run build -- --config grpc/src/main/resources/assets/pt/webpack.config.js
+# Bundle files needed for non-PT GUI
```

Comment:
I figured out that this is the correct way to get `main.js` built automatically in CI, which is what we want (so any changes to GUI code always actually "make it through" to the resulting docker image that gets built). This lets us delete `main.js`, which I also do in this PR